internal/store/store.go

```diff
@@ -445,18 +445,20 @@ func (s *Store) GetAllComposes() map[uuid.UUID]compose.Compose {
 	return composes
```

Comment:
General comment for the patch:
Very nice, but commit message should explain a bit more. Comment on (and in the future, even better if you had split the patch apart): which part is just a refactoring/renaming, what is the functional change (renaming the directory) and why is the image creation now necessary. All sort of obvious, but stating it in the commit message makes reviewing much easier.

Reply:
I think the size of this commit is kinda low and the renamings are obvious, therefore I think it's not worth splitting it up. I totally agree with the commit message, I think I got tired and omitted it, because as you said it's "all sort of obvious" :smile: I added a better commit message, feel free to propose more changes. :slightly_smiling_face: